src/GraphQL/Mapper/ContentImageAssetMapperStrategy.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace Ibexa\GraphQL\GraphQL\Mapper;
+
+use eZ\Publish\API\Repository\Values\Content\Field;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
+use eZ\Publish\Core\FieldType\ImageAsset;
+use EzSystems\EzPlatformGraphQL\GraphQL\DataLoader\ContentLoader;
+
+final class ContentImageAssetMapperStrategy implements ImageAssetMapperStrategyInterface
+{
+    /* @var \eZ\Publish\Core\FieldType\ImageAsset\AssetMapper */
+    private $assetMapper;
+
+    /** @var \EzSystems\EzPlatformGraphQL\GraphQL\DataLoader\ContentLoader */
+    private $contentLoader;
+
+    public function __construct(
+        ImageAsset\AssetMapper $assetMapper,
```

Comment:
Please use a proper import for `AssetMapper`.

Reply:
What's wrong with such import? It's definitely more readable than just `Value` and we are using such import throughout plenty of packages.